mac

```diff
@@ -341,7 +344,11 @@ install_dependencies() {
   fancy_echo "Installing dependencies"
   brew bundle --global --verbose --no-upgrade
 
-  setup_postgres
+  if [[ "$(which postgres)" =~ "postgres not found" ]] && [[ ! -d "/Applications/Postgres.app" ]]; then
```

Comment:
Why two conditional checks? Is `Postgres.app` an application installed by default (I don't have it)?

It might be better to create a function (like `fancy_echo`) to check if a package is installed:

```bash
if [[! is_installed postgres]]; then
```

Reply:
Instaling `/Applications/Postgres.app` is an edge case so let's remove it and have only the first check.

Reply:
Ok I will remove the `Postgres.app` check from the brew install `as well`. Do you want to leave the postgres check there prior to calling `setup_postgres` or leave this to another ticket, ie my issue from last year https://github.com/nimblehq/laptop/issues/35

Reply:
This improvement can be made in another PR. it is a nice-to-have at this point.